roles/command_line_tools/README.md

```diff
@@ -3,19 +3,21 @@ Xcode Command Line Tools
 
 Install Xcode command line tools.
 
+For macOS >= 12.0, installing the command line tools also installs a Python interpreter.
```

Comment:
I think this could be more precise:
```suggestion
For macOS >= 12.0, installing the command line tools also installs an Apple-packaged Python 3 interpreter, as a side-effect.
```

Reply:
Also, you could mention that it's worth to use `ansible_python_interpreter: /usr/bin/python3` with macOS 12.3+ since `/usr/bin/python` would be missing in this case.

Reply:
Yeah, I planned to update the docs once the functionality is fully complete.

Reply:
Ah, maybe. I haven't actually tested it. I just had to override it because of a global setting.